src/screens/SUE/SueReportScreen.tsx

```diff
@@ -164,15 +164,21 @@ export const SueReportScreen = ({
     setIsLoading(true);
     mutateAsync(formData)
       .then(() => {
-        setIsDone(true);
-        resetStoredValues();
+        setTimeout(() => {
+          setIsDone(true);
+          resetStoredValues();
+        }, 1500);
       })
       .catch(() => {
         setIsLoading(false);
 
         return Alert.alert(texts.defectReport.alerts.hint, texts.defectReport.alerts.error);
       })
-      .finally(() => setIsLoading(false));
+      .finally(() => {
+        setTimeout(() => {
+          setIsLoading(false);
```

Comment:
`catch` only works if an error occurs. `finally` always works. but I think it can be moved to `setIsLoading(false)`, `then` section. As a result, we set the `isLoading` value to `false` in `finally`. We can also do this in the `then` section. If an error occurs, `setIsDone` will never be `true`.

Reply:
we want this to happen for `then` and `catch` right, and if we place it explicitly in `catch` we can move it to `then` as well an do not need the `finally`, or am i wrong?

Reply:
Yes, that's right. As I just said, it can be moved to the `then` section. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1023/commits/35d639541e4ac609d683ea7f3a98741f68aa17ca